src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -329,6 +342,7 @@ export const useSendDetails = (): UseSendDetailsReturnType => {
   }, [estimatedFees, sendMax, setValue])
 
   const handleSendMax = useCallback(async () => {
+    isManualInputChange.current = false
```

Comment:
Ok this part I don't like it tbh, I hate using refs for such use cases, it means our pattern is wrong

I used it as a quick win because, when you were using `sendMax` for the first time, the handleInputChange was triggered, and so resetting `sendMax` to false...

Then if you clicked again, as the amount was not changing, it was good 😆